.osparc/jupyter-ml-pytorch/metadata.yml

```diff
@@ -0,0 +1,71 @@
+name: Jupyter ML PyTorch
+thumbnail: https://imgur.com/0ndE7uX.png
+description: "JupyterLab with PyTorch installed. No other scientific tools are provided."
+key: simcore/services/dynamic/jupyter-ml-pytorch
+version: 2.0.0
+integration-version: "2.0.0"
+type: dynamic
+authors:
+  - name: "Andrei Neagu"
+    email: "neagu@itis.swiss"
```

Comment:
quotes vs no quotes for fasse

Reply:
Sorry he is already here. I would keep him.

Reply:
I meant "Andrei Neagu" vs Alessandro Fasse. (The quotes)